data/web/inc/vars.inc.php

```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```

Comment:
Remember, this page is used by the end user, not that much by the admin. They might never have even heard about "groupware" or "ActiveSync". Also, ActiveSync is not used in the browser through the link here, so it doesn't need to be mentioned in the description.